### PR TITLE
Loading spinner

### PR DESCRIPTION
**Pull request by jakobw**
Task: https://phabricator.wikimedia.org/T120952
Demo: http://tools.wmflabs.org/file-reuse-test/loading_spinner/

Instruction: There is one loading spinner on the "Los!" button when submitting any image, another when loading a Wikipedia page (e.g. `https://de.wikipedia.org/wiki/Eichhörnchen`) when clicking on a picture and also one for the submit button on the feedback form.

### CODE REVIEW

File: js/app/Spinner.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+$.extend( Spinner.prototype, {
+	/**
+	 * @type {jQuery}
+	 */
+	_$node: null,
+
+	/**
+	 * Adds a spinner with `size` px height to a node.
+	 *
+	 * @param {int} size
```
Comment (wmde-manicki) [inline js/app/Spinner.js]: I am just curious but is this parameter actually required? it does not seem to be referenced to in the `add` functions body. And some calls of that functions pass some value of the parameter but i.e. when adding the spinner in the feedback form, there is no parameter.
I also removed those parameters where they are currently and it doesn't seem to affect the size of spinners (they look as good as before).

**Comment by wmde-manicki** [comment]
Tested in Firefox and Chromium and spinners look great!

File: js/app/Spinner.js (hunk 1 above)
Comment (jakobw) [inline js/app/Spinner.js]: Oops, I forgot that I decided to set the size in the CSS instead halfway through the implementation. Well spotted! Will change that in a bit.

Comment (jakobw) [inline js/app/Spinner.js]: Fixed

File: css/main.css [hunk 2]
```diff
@@ -689,3 +688,30 @@ a:hover {
...
+  font-size: 50px;
+  top: 50%;
+}
+@-moz-keyframes spin { 100% { -moz-transform: rotate(360deg); } }
+@-webkit-keyframes spin { 100% { -webkit-transform: rotate(360deg); } }
+@keyframes spin { 100% { -webkit-transform: rotate(360deg); transform:rotate(360deg); } }
+.spin-icon {
+  -webkit-animation: spin 2s linear infinite;
+  -moz-animation: spin 2s linear infinite;
+  animation: spin 2s linear infinite;
```
Comment (WMDE-Fisch) [inline css/main.css]: Don't want to be the party pooper again but this wont work with IE 8 & 9.  ( see http://caniuse.com/#feat=css-animation ) We could just ignore the fact ( 3% wont necessarily need a spinner ) - or use an animated image instead.

Comment (jakobw) [inline css/main.css]: Not sure if spinners that aren't moving for 3% of the users are really a problem. The fact that the entire application basically isn't working in IE8/9 at all makes an improvement like this pretty much pointless IMO.

Comment (WMDE-Fisch) [inline css/main.css]: Agree :-)

**Pull request merged**